eventhandlers/action_runners/actions/HuggingFaceZeroShot.py

```diff
@@ -34,78 +39,87 @@ def test_execute_action(self, session, file_id, connection_id):
         pass
 
     def execute_action(self, session, do_save_annotations=True) -> bool:
-        print("Hugging face has been executed")
-        return
-        """
-                    Creates a task from the given file_id in the given task template ID.
-                :return:
-                """
-
+        event_name = self.action.trigger_data.get('event_name')
         file_id = self.event_data['file_id']
-        if not file_id:
-            logger.warning(f'Action has no file_id Stopping execution')
-            return
-    
+        project_id = self.action.config_data['project_id']
+        group_id = self.action.config_data['group_id']
```

Comment:
group could mean anything, clarify attribute group
I'd like to learn more on plan for handling this type of "sub schema" selection - maybe something more general to talk about

Reply:
I agree that group_id isn't quite descriptive, the reason is it named that way is I was keeping the naming as it is on the other parts of the diffgram

Reply:
well then we should change those other parts too eventually
it should only be group if it's super clear it's in the attribute context